tests/unit/test_embedding_models/__init__.py

```diff
@@ -0,0 +1,5 @@
+# -*- coding: utf-8 -*-
```

Comment:
Let's stick to this headder 

https://meaningfy.atlassian.net/wiki/spaces/EP/pages/370671617/Python+header+template